app/models/application_setting.rb

```diff
@@ -0,0 +1,68 @@
+class ApplicationSetting
+  @defaults = begin
+                content = open(Rails.root.join('config', 'app.yml')).read
```

Comment:
__[RuboCop]__

Security/Open: The use of `Kernel#open` is a serious security risk.


<!-- Comment by [SideCI](https://sideci.com) -->